src/jit/inlinepolicy.h

```diff
@@ -216,13 +216,45 @@ class DiscretionaryPolicy : public LegacyPolicy
     const char* GetName() const override { return "DiscretionaryPolicy"; }
 
 private:
+    
+    void ComputeOpcodeBin(OPCODE opcode);
+    enum { MAX_ARGS = 4 };
```

Comment:
I don't think this is a big issue, but 2.3% is non-trivial and it seems that it could include some important methods to inline - I gather we don't know what % of that is 4 vs. 5+?